src/ancillary/anchor-request-params-parser.ts

```diff
@@ -139,13 +139,13 @@ export class AnchorRequestParamsParser {
   parse(req: ExpReq): Validation<RequestAnchorParams> {
     if (req.get('Content-Type') !== 'application/vnd.ipld.car') {
       // Legacy requests
-      Metrics.count(METRIC_NAMES.LEGACY_REQUESTED, 1)
+      Metrics.count(METRIC_NAMES.C_LEGACY_REQUESTED, 1)
```

Comment:
What does the `C_` prefix indicate?

Reply:
just controller - its to group together the controller ones so its not confusing with the service workers or scheduler

we could spell it out but i feel like super long names are not that efficient - if prefered could spell it out or put ctrl_ instead of c_

Reply:
anyway, if it was confusing once it will be again, changed it to 'ctrl_' @3benbox  ^

Reply:
maybe RC_ for RequestController?